guides/telemetry.md

```diff
@@ -0,0 +1,86 @@
+# Telemetry
+
+Absinthe 1.5 supports your use of `:telemetry` to instrument its activity.
+Call `:telemetry.attach/4` or `:telemetry.attach_many/4` to attach your
+handler function to any of the following event names:
+
+- `[:absinthe, :execute, :operation, :start]` when the operation starts
+- `[:absinthe, :execute, :operation]` when the operation finishes
+- `[:absinthe, :subscription, :publish, :start]` when a subscription starts
+- `[:absinthe, :subscription, :publish]` when a subscription finishes
+- `[:absinthe, :resolve, :field, :start]` when field resolution starts
+- `[:absinthe, :resolve, :field]` when field resolution finishes
+
+Only fields with a resolver get measured.
```

Comment:
Should probably be:
> By default, only fields with a resolver are measured.

Reply:
Oh, of course, I forgot people can add `meta: [absinthe_telemetry: true]` anywhere. Any other ways to override that default?